activestorage/test/analyzer/image_analyzer/image_magick_test.rb

```diff
@@ -70,6 +70,28 @@ class ActiveStorage::Analyzer::ImageAnalyzer::ImageMagickTest < ActiveSupport::T
     end
   end
 
+  test "when image_magick is not installed" do
+    old_value = ActiveStorage.const_get(:MINIMAGICK_AVAILABLE)
```

Comment:
You can use [stub_const](https://api.rubyonrails.org/classes/ActiveSupport/Testing/ConstantStubbing.html#method-i-stub_const) for this

Reply:
Thank you, I was looking for that but had to step away. Fixed!